pointillism-info.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index 565c78a..ce6160a 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -1,11 +1,18 @@
+ [Activity]
+ name = Pointillism
+-activity_version = 9
+-bundle_id = org.laptop.community.Pointillism
++activity_version = 9.1
++release_date = 2019-12-22
++metadata_license = CC0-1.0
++bundle_id = org.sugarlabs.Pointillism
+ exec = sugar-activity3 activity.Activity
+ show_launcher = yes
+ icon = activity
+ license = GPLv3+
+ summary = Create a pointillist image from the camera
++description = Pointillism is a Sugar activity to make photos in which small dots of color are applied in patterns to form an image.
+ repository = https://github.com/sugarlabs/pointillism.git
+-tags = Art
++tags = Graphics
++update_contact = tch@sugarlabs.org
++developer_name = Sugar Labs Community
++url = https://github.com/sugarlabs/pointillism-activity#pointillism
```

Comment:
it seems my edit got too late, it should be https://github.com/sugarlabs/pointillism-activity